addon/components/o-s-s/upload-area.stories.js

```diff
@@ -0,0 +1,195 @@
+import hbs from 'htmlbars-inline-precompile';
+import { action } from '@storybook/addon-actions';
+import { MockUploader } from 'dummy/controllers/application';
+
+const PrivacyTypes = ['public', 'private'];
+const SizeTypes = ['md', 'lg'];
+
+export default {
+  title: 'Components/OSS::UploadArea',
+  component: 'upload-area',
+  argTypes: {
+    uploader: {
+      type: { required: true },
+      description: 'The uploader instance. The uploader can extends BaseUploader class in service/base-uploader.ts',
+      table: {
+        type: {
+          summary: 'BaseUploader'
+        },
+        defaultValue: { summary: 'undefined' }
+      },
+      control: { type: null }
+    },
+    subtitle: {
+      description: 'The subtitle value.',
+      table: {
+        type: {
+          summary: 'string'
+        },
+        defaultValue: { summary: 'undefined' }
+      },
+      control: { type: 'text' }
+    },
+    disabled: {
+      description: 'If disabled, it will be impossible to upload file',
+      table: {
+        type: {
+          summary: 'boolean'
+        },
+        defaultValue: { summary: false }
+      },
+      control: { type: 'boolean' }
+    },
+    privacy: {
+      description:
+        'The privacy value of uploaded file. If the value is public, it can be visible by everyone.Else it can be visible by logged use',
```

Comment:
it will be visible by everyone, otherwise it will only be visible by logged-in users.